yb-voyager/src/dbzm/config.go

```diff
@@ -55,10 +58,8 @@ debezium.source.database.port=%d
 debezium.source.database.user=%s
 debezium.source.database.password=%s
 debezium.source.database.dbname=%s
```

Comment:
Didn't we move this to the base template?

Reply:
no, we moved `debezium.source.table.include.list` 
This can't be moved since it is different for MySQL i.e. `debezium.source.database.include.list`